src/types.rs

```diff
@@ -81,7 +81,7 @@ pub struct CompletionRequest<'a> {
     /// Controls diversity via nucleus sampling, not recommended to use with temperature
     pub top_p: f32,
     /// Controls the maximum number of tokens to generate in the completion
-    pub max_tokens: u32,
+    pub max_tokens: Option<u32>,
```

Comment:
Maybe add 
```rs
#[serde(skip_serializing_if = "Option::is_none")]
```
So we don't add additional null fields to requests? It [defaults to infinity anyways](https://platform.openai.com/docs/api-reference/chat/create#max_tokens)

Reply:
Yeah.  I'd better add.

Reply:
fixed https://github.com/Maxuss/chatgpt_rs/pull/66/commits/f4919560d6b9437567ae1f8e899108264cee3b18

Reply:
Fixed https://github.com/Maxuss/chatgpt_rs/pull/66/commits/f4919560d6b9437567ae1f8e899108264cee3b18